gatsby-config.js

```diff
@@ -57,6 +57,12 @@ module.exports = {
                 apiHost: process.env.GATSBY_SQUEAK_API_HOST,
             },
         },
+        {
+            resolve: `gatsby-mapbox-locations`,
+            options: {
+                mapboxToken: process.env.MAPBOX_TOKEN,
```

Comment:
My biggest concern is this.

I set this on Vercel already, I'm using a private token, but I'm happy to create a PostHog one. They have 50,000 requests/month for free, so we shouldn't hit that from production alone, but everytime we run this locally and/or create a preview environment, we'll fetch all of the latitude/longitudes. I'm pretty sure we'll eventually go over 50k/month if we do that. I'm not really sure it's worth it, though.

Other solutions:
- Just add a new latitude/longitude field to our Squeak profile and fetch it from there
- Use their enterprise plan and cache requests, should make it free IIUC?